apps/web/components/Embed.tsx

```diff
@@ -708,6 +708,32 @@ const EmbedTypeCodeAndPreviewDialogContent = ({
     removeQueryParams(router, ["dialog", ...queryParamsForDialog]);
   };
 
+  const themeSelector = () => (
```

Comment:
Why is it not being used as a Component? A component name starts with capital letter